chore: revamp readme (#16)
NabilSli/projet2: README.md

# 😃 BOOKI: Exemple of a travel agency website.
This website is my first achievement as a web developer👏. I did it during my boot camp at OpenClassroom's Web developer.

## 👀 Here you can check the final result
[See the web site](https://645a7889605f5a058dad93d1--incredible-shortbread-990e7e.netlify.app/#)

## ⭐️ Use the full range of knowledge acquired during the courses.
To realise this project there was some technical skill required, such as:

* 🎯 `HTML`
  * Build the page using the right tags

>   * dividing the page into blocks from the wireframe to ensure a responsive design

Reviewer: ```suggestion
  * Dividing the page into blocks from the wireframe to ensure a responsive design
```

  * Search engine optimization
  * code quality practices: comments and indent to make the code easily maintained and readable

>   * Use the right css class in order to not repeat the same code multiple time

Reviewer: ```suggestion
  * Factorize CSS class in order not to repeat the same code multiple time
```

                                                                           
* 📚 `CSS`

>   * Hierachise the code, starting witrh the desktop view as we are in a desktop first project, going top to bottom and from the bigger to the smaller view

Reviewer: ```suggestion
  * Hierachise the code, starting with the desktop view as we are in a desktop-first project, going top to bottom and from the bigger to the smaller view
```

# 😃 BOOKI: Exemple of a travel agency website.
This website is my first achievement as a web developer👏. I did it during my boot camp at OpenClassroom's Web developer.

## 👀 Here you can check the final result

> https://645a7889605f5a058dad93d1--incredible-shortbread-990e7e.netlify.app/#

Reviewer: Use an anchor please here 
```suggestion
[See the web site](https://645a7889605f5a058dad93d1--incredible-shortbread-990e7e.netlify.app/#)
```

Author: done :D

## ⭐️ Use the full range of knowledge acquired during the courses.
To realise this project there was some technical skill required, such as:

* 🎯 `HTML`

>   * Build the page using the right tabs

Reviewer: What the `right tabs` means ?

Author: i meant tags

Author: 🤣

>   * dividing the page in blocks from the wireframe to ensure a responsive design

Reviewer: ```suggestion
  * Dividing the page into blocks from the wireframe to ensure a responsive design
```

>   * Search engine optimization

Reviewer: Precise which one?

  * Good syntaxe, comments and indent to make the code easily maintened and readable

>   * Use css properties in order to not repeat the same code multiple time

Reviewer: Should be in the CSS part ?

Author: i was talking about css class in the html code

                                                                           
* 📚 `CSS`

>   * Hierachise the code

Reviewer: Add more detail; which pattern did you follow?

>   * No repetion

Reviewer: ```suggestion
  * Use CSS properties in order to not repeat the same code multiple time
```

Reviewer: You forgot this ^^

  * Use media query's to ensure reponsive design
  * Desktop first focus for this project
  * Use flexbox and/or grid

* 👨‍👩‍👧‍👧 `GIT`
  * Use git for vesrioning
  * Make pullrequest and have reviews of your work

>   * resolve issues

Reviewer: ```suggestion
  * Use `git` for versioning
  * Create branches and open pull requests for each new feature
  * Get reviewed and apply proper corrections
```
